hoodie-client/src/test/java/com/uber/hoodie/table/TestMergeOnReadTable.java

```diff
@@ -660,6 +662,76 @@ public void testLogFileCountsAfterCompaction() throws Exception {
     }
   }
 
+  @Test
+  public void testMetadataValuesAfterInsertUpsertAndCompaction() throws Exception {
```

Comment:
can we write a smaller test for testing if the non-serialized fields dont get serialized? We can just create a fake CommitMetadata object right?

Reply:
We can add a TestHoodieCommitMetadata or something?